TodoTasker/app/src/main/java/com/backend/todo_tasker/MainActivity.kt

```diff
@@ -35,6 +42,42 @@ class MainActivity : AppCompatActivity() {
 
         dbClass = DatabaseClass(applicationContext)
         todoDb = dbClass.createDb()
+
+        //DatePicker Code
+        val dateInputEditText = findViewById<EditText>(R.id.edittext_datetime)
+        dateInputEditText.setOnClickListener {
```

Comment:
Bitte in eigene Funktion auslagern. Die kannst dann im Layout angeben siehe EditText. Vl noch a klasse für die funktionsaufrufe.